ipyparallel/engine/engine.py

```diff
@@ -66,7 +67,23 @@ def tunnel_mod(self):
     # not configurable:
     connection_info = Dict()
     user_ns = Dict()
-    id = Integer(allow_none=True)
+    id = Integer(None, allow_none=True, config=True,
+        help="""Request this engine ID.
+        
+        If run in MPI, will use the MPI rank.
+        Otherwise, let the Hub decide what our rank should be.
+        """
+    )
+    @default('id')
+    def _id_default(self):
+        try:
+            from mpi4py import MPI
```

Comment:
I can definitely do that, but I'd rather not unless there is a reason. Is importing mpi4py ever harmful? It works fine if no MPI environment is present, so it would have to be some MPI use case where importing it does the wrong thing.